packages/forms-web-app/src/views/interested-party-guide/have-your-say-examination.njk

```diff
@@ -0,0 +1,99 @@
+{% extends "layouts/layout.njk" %}
+
+{% block pageTitle %}
+  Have your say during the examination of the application
+{% endblock %}
+
+{% block header %}
+  {{ pinsHeader() }}
+{% endblock %}
+
+{% block beforeContent %}
+
+  {% from "govuk/components/phase-banner/macro.njk" import govukPhaseBanner %}
+  {{ govukPhaseBanner({
+    tag: {
+      text: "alpha"
+    },
+    html: 'This is a new service – your <a class="govuk-link" href="#">feedback</a> will help us to improve it.'
+  }) }}
+
+{% endblock %}
+{% block content %}
+
+  <h1 class="govuk-heading-xl">
+        Have your say during the examination of the project
+    </h1>
+  <h2 class="pins-contents-list-title govuk-heading-s">Contents</h2>
+  <ol class="govuk-list pins-contents-list">
+    <li>
+      <a class="govuk-link" href="#what-examination-means">What examination means</a>
```

Comment:
Should be What Examination means ( with capital E )

Reply:
Fixed